app/src/main/res/values/strings.xml

```diff
@@ -289,4 +289,8 @@
     <string name="topAppBar_back">Back</string>
     <string name="utils_login_first">Login first</string>
     <string name="voteToggle">Toggle vote</string>
```

Comment:
Didn't drop this because I'm not sure of the implications since it's been translated a bunch

Reply:
If they aren't used, they should be deleted including the translations imo. Would need a code owner to make the final verdict.

Reply:
If its unused and not expected to be used again we should remove it, but it doesn't have to be in this PR necessarily.

Reply:
- [ ] Drop `voteToggle` unused string resource